### PR TITLE
Implement Accessibility Feature

### PR DESCRIPTION
**Pull request by EricDw**

### CODE REVIEW

File: sdk/src/main/kotlin/io/rover/sdk/ui/blocks/image/ViewImage.kt [hunk 1]
```diff
@@ -25,7 +26,19 @@ internal class ViewImage(
...
             val measuredSize = binding.measuredSize ?: throw RuntimeException("ViewImage may only be used with a view model binding including a measured size (ie. used within a Rover screen layout).")
 
-            imageView.alpha = 0f
+            with(imageView) {
+
+                alpha = 0f
+                val isAccessible = binding.viewModel.isClickable || !binding.viewModel.isDecorative
+                isFocusableInTouchMode = isAccessible
+                importantForAccessibility = if (isAccessible) {
+                    imageView.contentDescription = binding.viewModel.accessibilityLabel
```
Comment (samwoodall) [inline sdk/src/main/kotlin/io/rover/sdk/ui/blocks/image/ViewImage.kt]: [Super Nitpick] Could be `contentDescription = binding.viewModel.accessibilityLabel` like you're doing with alpha but ignore this if you're doing it to be more explicit about what you're setting contentDesc on

**Review by samwoodall — APPROVED** [review]
Great Stuff 👍

**Comment by samwoodall** [comment]
Just ran against an experience on staging, LGTM

**Review by samwoodall — CHANGES_REQUESTED** [review]
We need to change some of the `ImageOptimizationServiceSpec` code to account for the image model change. The tests won't compile because aren't passing all the required args to instantiate an `Image`,  we're missing `isDecorative` and `accessibilityLabel`

**Comment by EricDw** [comment]
@samwoodall Are we ready to merge this?

**Comment by samwoodall** [comment]
Yep

**Review by samwoodall — APPROVED** [review]

**Pull request merged**